tests/template_tests/syntax_tests/test_querystring.py

```diff
@@ -20,6 +20,17 @@ def test_querystring_empty_get_params(self):
             "test_querystring_empty_get_params", context, expected=""
         )
 
+    @setup({"test_querystring_remove_all_params": "{% querystring a=None %}"})
+    def test_querystring_remove_all_params(self):
+        non_empty_context = RequestContext(self.request_factory.get("/?a=b"))
+        empty_context = RequestContext(self.request_factory.get("/"))
+        self.assertRenderEqual(
+            "test_querystring_remove_all_params", non_empty_context, expected="?"
+        )
+        self.assertRenderEqual(
+            "test_querystring_remove_all_params", empty_context, expected=""
+        )
```

Comment:
Could you use `subTest` for these two asserts?
```suggestion
        non_empty_context = RequestContext(self.request_factory.get("/?a=b"))
        empty_context = RequestContext(self.request_factory.get("/"))
        for context, expected in [(non_empty_context, "?"), (empty_context, "")]:
            with self.subTest(expected=expected):
                self.assertRenderEqual("test_querystring_remove_all_params", context, expected)
```